.github/pull_request_template/bugfix.md

```diff
@@ -0,0 +1,26 @@
+# 🐛 Bug Fix
+
+## Checklist
+- [ ] Issue: #___
+- [ ] Branch: `patch-[issue]-[desc]`
+- [ ] Title: `<component>: <message> Fix #<issue>`
+- [ ] Added regression test
```

Comment:
I would say this instead

```suggestion
- [ ] Added regression tests
```

Just to indicate that we don't necessarily just want a single test, but as many as needed